### ISSUE TITLE
Optimization for Deep Learning Highlights in 2017

### ISSUE THREAD

**Issue by YouheiKomakine** [YouheiKomakine/articles#41]
source:
https://twitter.com/seb_ruder/status/937416428122189824

Sebastian Ruder@seb_ruder
New blog post: Optimization for Deep Learning Highlights in 2017

---
### [Optimization for Deep Learning Highlights in 2017](http://ruder.io/deep-learning-optimization-2017/index.html)
>Table of contents:
 - Improving Adam
   - Decoupling weight decay
   - Fixing the exponential moving average
 - Tuning the learning rate
 - Warm restarts
   - SGD with restarts
   - Snapshot ensembles
   - Adam with restarts
 - Learning to optimize
 - Understanding generalization
